static/app/components/sidebar/index.tsx

```diff
@@ -91,6 +91,12 @@ function Sidebar({location, organization}: Props) {
     return () => bcl.remove('body-sidebar');
   }, [bcl]);
 
+  useEffect(() => {
+    if (location?.hash === '#whats-new') {
+      togglePanel(SidebarPanelKey.Broadcasts);
+    }
+  }, [location?.hash]);
```

Comment:
Want to generalize it be 
```tsx
type SidebarHash = `#sidebar-${SidebarPanelKey}`;
```

and then you can parse the string and toggle whatever with `#sidebar-whats-new`

Reply:
@evanpurkhiser good call